modules/pxp-module-puppet

```diff
@@ -172,8 +172,14 @@ def run(params_and_config)
   config = params_and_config["config"]
 
   if config["puppet_bin"].nil? || config["puppet_bin"].empty?
-    return get_result_from_report(DEFAULT_ERROR_CODE, config,
+    if !is_win?
+      return get_result_from_report(DEFAULT_ERROR_CODE, config,
                                   "puppet_bin configuration value not set")
+    else
+      module_path = File.expand_path(File.dirname(__FILE__))
+      puppet_bin = File.join(module_path, '..', '..', 'bin', 'puppet.bat')
+      config["puppet_bin"] = File.expand_path(puppet_bin)
```

Comment:
I'm not sure I understand your comment.  This is a fallback value when `puppet_bin` hasn't been specified in the config file.